local_minio/minio_install.sh

```diff
@@ -36,41 +39,49 @@ function do_install {
 
   
   if [ ! -f /usr/bin/minio ]; then
-    curl -o /usr/bin/minio "http://dl.min.io/server/minio/release/linux-$platform/minio"
+    echo "Downloading release"
+    curl -L -o /usr/bin/minio "http://dl.min.io/server/minio/release/linux-$platform/minio"
     chmod +x /usr/bin/minio
   fi
   sed -i "s/#host_number#/$array_len/" local_minio/minio.service
   sed -i "s/#group_id#/$GROUPID/g" local_minio/minio.service
   cp local_minio/minio.service /lib/systemd/system/minio.service
   chmod 664 /lib/systemd/system/minio.service
   systemctl daemon-reload
+  echo "Service minio start"
   service minio start
+  echo "Service avahi-daemon restart"
   service avahi-daemon restart
 }
 
 function uninstall_only {
   #stop services
+  echo "Stoping Services"
   service minio stop
   service minio_lookup stop
 
   #remove service files
+  echo "Removing services files"
   rm /lib/systemd/system/minio.service
   rm /lib/systemd/system/minio_lookup.service
   
   #remove data
+  echo "Removing data"
   rm -rfR /data/$GROUPID/
   
   #remove support files
+  echo "Removing support files"
   rm /usr/bin/service_lookup.py
   rm /usr/bin/minio
   systemctl daemon-reload
   
   #clean hosts file
+  echo "Cleaning host files"
```

Comment:
Let's remove the comments above the echo commands and add an echo at the very end saying "Uninstallation Successful"